openshift_performance/ci/scripts/large_object_list/large_object_list.sh

```diff
@@ -0,0 +1,176 @@
+###############################################
+## Auth=lhorsley@redhat.com
+## Description: Cluster resilience testing via creation of a large number of secrets with a specific labelSelector in namespaces on an OWN cluster
+## Polarion test case: OCP-41643 - Large object list testing (Listing secrets in all namespaces with a specific labelSelector)	
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-65206
+## Bugs related: https://bugzilla.redhat.com/show_bug.cgi?id=2050230 and https://bugzilla.redhat.com/show_bug.cgi?id=2094012
+## Cluster config: 3 master (m5.4xlarge or equivalent) with 40 workers (vm_type_workers: m5.2xlarge or equivalent).
+## The machine running the test should have at least 4 cores.
+## Note: While the test runs, check the functionality of the cluster. A simple example:
+##       while true; do oc get co --no-headers| grep -v 'True.*False.*False'; oc get nodes --no-headers| grep -v ' Ready'; date; sleep 10; done
+################################################ 
+
+#! /bin/bash
+
+
+
+expected_namespaces=250
+expected_secrets=5000
+secrets_per_namespace=20
```

Comment:
1. Can we make these as parameters or overwrite by env variables? 
2. expected_secrets can be calculated the the expected_namespaces x secrets_per_namespace in code. So we only need 2 inputs.